.devcontainer/contributor/devcontainer.json

```
@@ -0,0 +1,31 @@
+{
+    "name": "Radius CLI - Contributor",
+    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {
+            "version": "1.21"
+        },
+        "ghcr.io/guiyomh/features/golangci-lint:0": {},
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {
+            "minikube": "none"
+        },
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {
+            "version": "latest",
+            "moby": true
+        }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "redhat.vscode-yaml",
+                "streetsidesoftware.code-spell-checker",
+                "golang.go",
+                "ms-azuretools.vscode-bicep",
```

```suggestion
                "ms-azuretools.rad-vscode-bicep",
```

Until we converge on the official Bicep compiler, this needs to be our fork.